README.md

```diff
@@ -1 +1,181 @@
+
+> Ruby
+  Postgres SQL
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> 
+
+- [Live Demo Link](UNDER-CONSTRUCTION)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- [ ]  install Ruby
+- [ ]bundle init
+- [ ]bundle install
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- [ ] Open terminal
+- [ ] git clone https://github.com/monicah79/Capstone-ruby.git
+- [ ] cd Capstone-ruby
+  
+
+### Install
+
+Install this project with:
+
+ -[ ] Install Ruby
+ -[ ] bundle install
+ -[ ] gem install
+
+
+
+### Run tests
+
+To run tests, run the following command:
+
+-[] rspec spec
+
+
+### Deployment
+
+You can deploy this project using:
+
+ -[] COMING SOON
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Monicah**
+
+- GitHub: [@monicah79](https://github.com/monicah79)
+- Twitter: [@MonicahWanjema](https://twitter.com/MonicaWanjema)
+- LinkedIn: [monicah-wanjema](https://www.linkedin.com/in/monicah-wanjema/)
+
+👤 **Author2**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[Save data on database]**
+- [ ] **[Test with rspec spec]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/monicah79/Capstone-ruby/issues).
+
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+- Give a ⭐️ if you like this project!
+
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+-  I would like to thank [Microverse](https://www.microverse.org/) for giving me this opportunity to learn and grow as a developer.
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Hi @monicah79,  please fix the broken 'LICENCE' link in the README file that appears to be broken. When attempting to access it, a 404 error page is displayed.
![image](https://user-images.githubusercontent.com/101400298/231146753-1b40106b-9a02-4ce2-930e-ae935845af84.png)